src/components/Videos.vue

```diff
@@ -121,6 +122,211 @@ export default {
+				gu: 'Gujarati',
+				gv: 'Manx',
+				ha: 'Hausa',
+				he: 'Hebrew',
+				hi: 'Hindi',
+				ho: 'Hiri Motu',
+				hr: 'Croatian',
+				ht: 'Haitian',
+				hu: 'Hungarian',
+				hy: 'Armenian',
+				hz: 'Herero',
+				ia: 'Interlingua',
+				id: 'Indonesian',
+				ie: 'Interlingue',
+				ig: 'Igbo',
+				ii: 'Sichuan Yi',
+				ik: 'Inupiaq',
+				io: 'Ido',
+				is: 'Icelandic',
+				it: 'Italian',
+				iu: 'Inuktitut',
+				ja: 'Japanese',
+				jv: 'Javanese',
+				ka: 'Georgian',
+				kg: 'Kongo',
+				ki: 'Kikuyu',
+				kj: 'Kuanyama',
+				kk: 'Kazakh',
+				kl: 'Kalaallisut',
+				km: 'Central Khmer',
+				kn: 'Kannada',
+				ko: 'Korean',
+				kr: 'Kanuri',
+				ks: 'Kashmiri',
+				ku: 'Kurdish',
+				kv: 'Komi',
+				kw: 'Cornish',
+				ky: 'Kirghiz',
+				la: 'Latin',
+				lb: 'Luxembourgish',
+				lg: 'Ganda',
+				li: 'Limburgan',
+				ln: 'Lingala',
+				lo: 'Lao',
+				lt: 'Lithuanian',
+				lu: 'Luba-Katanga',
+				lv: 'Latvian',
+				mg: 'Malagasy',
+				mh: 'Marshallese',
+				mi: 'Maori',
+				mk: 'Macedonian',
+				ml: 'Malayalam',
+				mn: 'Mongolian',
+				mr: 'Marathi',
+				ms: 'Malay',
+				mt: 'Maltese',
+				my: 'Burmese',
+				na: 'Nauru',
+				nb: 'Norwegian Bokmål',
+				nd: 'Ndebele, North',
+				ne: 'Nepali',
+				ng: 'Ndonga',
+				nl: 'Dutch',
+				nn: 'Norwegian Nynorsk',
+				no: 'Norwegian',
+				nr: 'Ndebele, South',
+				nv: 'Navajo',
+				ny: 'Nyanja',
+				oc: 'Occitan',
+				oj: 'Ojibwa',
+				om: 'Oromo',
+				or: 'Oriya',
+				os: 'Ossetian',
+				pa: 'Panjabi',
+				pi: 'Pali',
+				pl: 'Polish',
+				ps: 'Pushto',
+				pt: 'Portuguese',
+				qu: 'Quechua',
+				rm: 'Romansh',
+				rn: 'Rundi',
+				ro: 'Romanian',
+				ru: 'Russian',
+				rw: 'Kinyarwanda',
+				sa: 'Sanskrit',
+				sc: 'Sardinian',
+				sd: 'Sindhi',
+				se: 'Northern Sami',
+				sg: 'Sango',
+				si: 'Sinhala',
+				sk: 'Slovak',
+				sl: 'Slovenian',
+				sm: 'Samoan',
+				sn: 'Shona',
+				so: 'Somali',
+				sq: 'Albanian',
+				sr: 'Serbian',
+				ss: 'Swati',
+				st: 'Sotho',
+				su: 'Sundanese',
+				sv: 'Swedish',
+				sw: 'Swahili',
+				ta: 'Tamil',
+				te: 'Telugu',
+				tg: 'Tajik',
+				th: 'Thai',
+				ti: 'Tigrinya',
+				tk: 'Turkmen',
+				tl: 'Tagalog',
+				tn: 'Tswana',
+				to: 'Tonga',
+				tr: 'Turkish',
+				ts: 'Tsonga',
+				tt: 'Tatar',
+				tw: 'Twi',
+				ty: 'Tahitian',
+				ug: 'Uighur',
+				uk: 'Ukrainian',
+				ur: 'Urdu',
+				uz: 'Uzbek',
+				ve: 'Venda',
+				vi: 'Vietnamese',
+				vo: 'Volapük',
+				wa: 'Walloon',
+				wo: 'Wolof',
+				xh: 'Xhosa',
+				yi: 'Yiddish',
+				yo: 'Yoruba',
+				za: 'Zhuang',
+				zh: 'Chinese',
+				zu: 'Zulu',
+			}
+			const langs = this.$attrs.caplangs
+			let tracks = ''
+			// Create caption tracks for the HTML5 player
+			// E.g.: movie.mkv --> movke.en.vtt
+			for (let i = 0; i < langs.length; ++i) {
+				const lang = langs[i]
+				let language = languages[lang]
+				if (language.length === 0) language = lang
+				tracks += '<track src="'
```

Comment:
It might be better to create these elements with vue (`v-for`) and  not by creating a string here.

Reply:
Thanks a lot @beardhatcode!
I'll have a look at the 'v-for' syntax and try and use it.
I'll be updating my code and re-submitting.

Also, I may add the possibility to use '.movie.en.vtt' for the language tracks -- with a leading dot.
So as not to clutter the contents of the viewer folder with language tracks.

Reply:
Ok, that could work but in that case you could have two vtt files per language be sure to give precedence to the visible file (without a dot). However, I would first try to just get it working and leave this addition for another PR.

Reply:
OK